### ISSUE TITLE
Font size messed up

### ISSUE THREAD

**Issue by gaetgu** [rougier/svg-tag-mode#38]
I am trying to get svg-tags set up with org mode (using org-modern and doom emacs). However, I am having some issues. Here is what the tags look like:

<img width="852" alt="image" src="https://user-images.githubusercontent.com/47652746/207126676-d110d7ac-2d6c-4682-92f7-98a690374d30.png">

I have looked through some of the other issues (mostly #12 and #10) and I have tried using the fork from PR #14, with no success. I am wondering if there is anything obvious that I am missing?

**Comment by rougier** [comment]
No, it's a recurrent problem that I've hard time fixing. Do you use any kind of magnification on your os?

**Comment by ronisbr** [comment]
I have the exactly the same problem in macOS with default configurations of monitor.

**Comment by rougier** [comment]
@ronisbr Could you tr the various fixes pointed by @gaetgu (#12 and #10) and the fork?

**Comment by ronisbr** [comment]
Hi @rougier !

I found the problem here! Everything started to work if I change this line:

```lisp
         (font-info       (font-info (format "%s-%d" font-family font-size)))
```

to

```lisp
         (font-info       (font-info "JetBrainsMono Nerd Font Mono"))
```

The problem is that the name of my font does not appear to follow the expected format!

**Comment by ronisbr** [comment]
After the modification:

<img width="331" alt="Captura de Tela 2023-03-21 às 12 34 43" src="https://user-images.githubusercontent.com/1068295/226657906-621e250e-7888-4628-9a80-bbca6db1d22b.png">

**Comment by rougier** [comment]
Fantastic! Thanks for the very helpful debug! This size problem has plagued the library from the start without any obvious solution. Now the question is why do I not pick the right size when required? Do you know when your default font size is actually set?

**Comment by ronisbr** [comment]
Hi @rougier !

I have no idea. Since I use Doom Emacs, it has its own method to customize the default fonts. This is the code where I set the font in Doom: https://github.com/ronisbr/doom.d/blob/25c12efa4dc90ca2b6b2824047c33fc4f2b596d4/settings/setup-theme.el#L3 It is a variable that is used to set Emacs fonts at some stage.

If we manage to find a hook that runs after that, we might just add comments to the README here instructing the users to add a specific function to that hook setting the fonts in svg-tag-mode.

**Comment by rougier** [comment]
I see. Then maybe a warning/howto in the README would be our best option since I don't think it's possible to have a hook on default font change.

**Comment by tam5** [comment]
Re: above regarding hooks and such, for temp fix I'm doing the following:

```elisp
(use-package! svg-lib
  :init (add-hook 'after-setting-font-hook (lambda () (setq svg-lib-style-default (svg-lib-style-compute-default)))))
```

Which seems to alleviate the issue for me.

Will try to add more debug info i've found soon

**Comment by rougier** [comment]
Oh, the `after-setting-font-hook` is really useful. Maybe we could include your fix in the lib.
Could you make a PR (for this and svg-lib as well since it may suffer from the same problem)
